### ISSUE TITLE
thumbnails.libretro.com is down

### ISSUE THREAD

**Issue by ngokevin** [libretro-thumbnails/libretro-thumbnails#145]
The thumbnail server is down!

https://github.com/spruceUI/spruceOS/issues/259

**Comment by RobLoach** [comment]
Thanks. There's conversations about this in the Discords. Stay tuned.

**Comment by Hoff16** [comment]
Has there been any update here? Looks like the server is back up but thumbnail downloads in retroarch is not.

**Comment by RobLoach** [comment]
Still working on it.

**Comment by Hoff16** [comment]
> Still working on it.

Awesome, good to know. Thanks!

**Comment by Hoff16** [comment]
Looks like this has been partially fixed or in process? Is that right?

**Comment by RobLoach** [comment]
Correct. I believe what's missing is the cronjob to keep it up to date. Join on Discord if you're looking for more deets.
